src/app/shared/models/action.enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum ActionEnum {
+  Block = 'lock',
+  Archivate = 'archive',
+  Edit = 'edit',
+}
```

Comment:
enum should be in constants. I think this shoild have a place in andmin module. It should not have enum in the name. Should we specify that it is equipment actions?